baseplate/clients/thrift.py

```diff
@@ -162,22 +167,53 @@ def retrying(self, **policy: Any) -> Iterator["_PooledClientProxy"]:
         )
 
 
+PROM_NAMESPACE = "thrift_client"
+
+PROM_COMMON_LABELS = [
+    "namespace",
```

Comment:
not sure exactly what this is since I've never used Thrift, is this akin to `grpc_service`? Happy to add it to the spec if it was a miss

Reply:
The equivalent tag is called `slug` below, so I guess it's the name of the client? Deleting it.